### PR TITLE
Joshua s brown/include formating

### PR DESCRIPTION
**Pull request by JoshuaSBrown**
Updating the include formatting 
https://github.com/votca/votca/issues/300

### CODE REVIEW

File: include/votca/xtp/IndexParser.h [hunk 1]
```diff
@@ -21,9 +21,12 @@
 #ifndef VOTCA_XTP_INDEXPARSER_H
 #define VOTCA_XTP_INDEXPARSER_H
 
+// Standard includes
 #include <string>
 #include <vector>
-#include <votca/tools/types.h>
+
+// VOTCA includes
+#include "votca/tools/types.h"
```
Comment (JensWehner) [inline include/votca/xtp/IndexParser.h:29]: this should not be?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,7 @@ For more detailed information about the changes see the history of the
 [repository](https://github.com/votca/xtp/commits/master).
 
 ## Version 1.7-dev
+* Standardising include order and style in header files (#448)
```
Comment (junghans) [inline CHANGELOG.md:5]: Should go at the end of the block.

File: include/votca/xtp/ERIs.h [hunk 3]
```diff
@@ -21,8 +21,9 @@
 #ifndef VOTCA_XTP_ERIS_H
 #define VOTCA_XTP_ERIS_H
 
-#include <votca/xtp/fourcenter.h>
-#include <votca/xtp/threecenter.h>
+// Local VOTCA includes
+#include "votca/xtp/fourcenter.h"
+#include "votca/xtp/threecenter.h"
```
Comment (junghans) [inline include/votca/xtp/ERIs.h:26]: Should you drop the `votca/xtp` prefix as this file is on the same folder?

Comment (JensWehner) [reply include/votca/xtp/ERIs.h:26]: yes good point

Comment (JensWehner) [reply include/votca/xtp/ERIs.h:26]: @junghans maybe we should also be more selective in which headers we install and move most to src? Any opinions?

Comment (junghans) [reply include/votca/xtp/ERIs.h:26]: Yes, I am all for it.

File: include/votca/xtp/IndexParser.h (hunk 1 above)
Comment (JoshuaSBrown) [reply include/votca/xtp/IndexParser.h:29]: Good catch

File: include/votca/xtp/ERIs.h (hunk 3 above)
Comment (JoshuaSBrown) [reply include/votca/xtp/ERIs.h:26]: Sure, but I wouldn't know which ones to move? I guess for xtp we could start by moving all of them, are there any tools that are built using xtp?

**Comment by JoshuaSBrown** [comment]
[Where to put private headers](https://github.com/votca/votca/issues/303)

File: include/votca/xtp/ERIs.h (hunk 3 above)
Comment (JoshuaSBrown) [reply include/votca/xtp/ERIs.h:26]: > Shouldn't you drop the `votca/xtp` prefix as this file is in the same folder?

Ok

Comment (JoshuaSBrown) [reply include/votca/xtp/ERIs.h]: > Shouldn't you drop the `votca/xtp` prefix as this file is in the same folder?

From what I have read it actually appears that leaving the prefix is a safer convention.

Comment (junghans) [reply include/votca/xtp/ERIs.h]: There are two issues, the way how headers include each other and which ones are public headers.

File: include/votca/xtp/cudamatrix.h [hunk 4]
```diff
@@ -17,20 +17,26 @@
  *
  */
 
-#ifndef __XTP_CUDA_MATRIX__H
-#define __XTP_CUDA_MATRIX__H
+#ifndef VOTCA_XTP_CUDAMATRIX_H
+#define VOTCA_XTP_CUDAMATRIX_H
 
+// CMake generated file
 #include <votca/xtp/votca_config.h>
```
Comment (junghans) [inline include/votca/xtp/cudamatrix.h:24]:
```suggestion
#include "votca_config.h"
```

File: include/votca/xtp/cudapipeline.h [hunk 5]
```diff
@@ -17,15 +17,17 @@
  *
  */
 
-#ifndef __XTP_CUDA_PIPELINE__H
-#define __XTP_CUDA_PIPELINE__H
+#ifndef VOTCA_XTP_CUDAPIPELINE_H
+#define VOTCA_XTP_CUDAPIPELINE_H
 
+// CMake generated file
 #include <votca/xtp/votca_config.h>
```
Comment (junghans) [inline include/votca/xtp/cudapipeline.h:24]:
```suggestion
#include "votca_config.h"
```

File: include/votca/xtp/eigen.h [hunk 6]
```diff
@@ -21,9 +21,13 @@
 #ifndef VOTCA_XTP_EIGEN_H
 #define VOTCA_XTP_EIGEN_H
 
+// CMake Generated file
+#include <votca/xtp/votca_config.h>
```
Comment (junghans) [inline include/votca/xtp/eigen.h:25]:
```suggestion
#include "votca_config.h"
```

**Comment by JoshuaSBrown** [comment]
Why is travis hanging...

**Comment by JoshuaSBrown** [comment]
@JensWehner Is this behavior desired? https://www.deepcode.ai/app/gh/votca/xtp/d240a77249118c5b9ab4c50d840c19ca02e48b49/votca/xtp/b3fca97596494f2924daf93a5cc296b1942f8c89/pr/_/%2F/code

**Comment by JensWehner** [comment]
I have no idea what this means. The code looks okay.

**Comment by JoshuaSBrown** [comment]
> Why is travis hanging...

Looks like the issue comes from including comments. 
https://github.com/travis-ci/travis-ci/issues/785

**Pull request merged**